packages/backend/apps/finances/signals.py

```diff
@@ -10,4 +11,9 @@
 @receiver(post_save, sender=User)
 def create_free_plan_subscription(sender, instance, created, **kwargs):
     if created:
-        subscriptions.initialize_user(user=instance)
+        try:
+            subscriptions.initialize_user(user=instance)
+        except AuthenticationError:
+            return
```

Comment:
Maybe it would be a good idea to handle this error by at least sending this error to stderr (or sentry etc)